Dockerfile

```diff
@@ -285,7 +285,7 @@ RUN \
 
 ARG HOST_ARCH
 ENV VENDOR="bottlerocket"
-ENV RUSTVER="1.85.1"
+ENV RUSTVER="1.86.0"
```

Comment:
Do you need to keep the commits that bump 1.85.0 -> 1.85.1 -> 1.86.0 -> 1.87.0?

Reply:
I kept them to show the progression, and in case we needed to revert 1.87.0 at the last minute without necessarily backing out the other changes. 1.85.1 is useful as it lets us drop the `cc-rs` patch.